### PR TITLE
Accessible mega nav

### PR DESCRIPTION
**Pull request by frrrances**
This PR makes the main mega nav for the libraries more accessible by adding structure, aria labels and controls, and some keyboard functionality for opening and closing the menus.

### CODE REVIEW

**Comment by frrrances** [comment]
This branch is up on dev right now.

File: footer.php [hunk 1]
```diff
@@ -130,6 +132,49 @@
...
+        var $focused = document.activeElement;
+
+        switch(e.keyCode)
+        {
+            // user presses the "space"
+            case 32:    showMenu(e, $focused);
+                        break;
+
+            // user presses the "esc" key
+            case 27:    hideMenu(e,);
```
Comment (matt-bernhardt) [inline footer.php]: This line is throwing a syntax error in Firefox and IE on Windows. `expected expression, got ')'`

Comment (frrrances) [inline footer.php]: Nice catch! Fixed it... let me know if that helps things.

**Comment by frrrances** [comment]
@matt-bernhardt Thanks for the good review. I addressed the extra comma, the subnav showing on mobile menu hover issue, and fixed the tests. Let me know if there's anything else I should address.

**Comment by frrrances** [comment]
@matt-bernhardt  I moved the js into the grunt files. Let me know if this looks good.

**Pull request merged**